lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -13,6 +13,7 @@
   // so maybe we add the border-radius to all media-* classes?
   & .media-video {
     border-radius: 4px;
+    padding-bottom: 60.4%;
```

Comment:
Hmm, can you add a comment for this? Also makes me wonder if we make this calculation a mixin.